gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -745,9 +745,8 @@ private void cancelCurrentRequest() {
       requestContext.close();
       requestContext = null;
     }
-    if (resIterator != null) {
-      resIterator = null;
-    }
+    drainIterator();
```

Comment:
nit: I know that `resIterator` is a global/object variable. It will be unambiguous if we can pass `resIterator` as parameter to `drainIterator`

Reply:
I don't think it makes a difference for private method, rather it would be redundant. I would agree for any public api. 
`resIterator` is not global variable, it is an instance variable here.